admin-management-service/src/main/java/com/scaler/adminmanagementservice/models/AdminModel.java

```diff
@@ -0,0 +1,7 @@
+package com.scaler.adminmanagementservice.models;
+
+public class AdminModel {
+    private Long id;
```

Comment:
how are you going to access the private fields

Reply:
we can access via getter setter when we use it , will annotate it by @Getter and @Setter